src/Weebly/WeeblyClient.php

```diff
@@ -247,7 +247,11 @@ private function makeRequest($url, $parameters=array(), $method='POST')
     {
         $curl_handler = $this->getCurlHandler();
 
-        if ($method !== 'GET'){
+        if ($method === 'GET'){
+            // The HTTP method may have been set to POST. Reset the POST options on the handler to default to GET.
+            curl_reset($curl_handler);
+            $options = [];
```

Comment:
What happens to the `$parameters` in a GET call? They aren't passed along at all?